src/Lean/Server/Utils.lean

```diff
@@ -66,23 +67,11 @@ namespace Lean.Lsp.DocumentUri
 
 /-- Transform the given path to a file:// URI. -/
 def ofPath (fname : System.FilePath) : DocumentUri :=
```

Comment:
We can probably just rm these two now.

Reply:
I see, it looks like the previous version returns `none` when the argument is not actually a `file://` URI and yours returns the argument in that case. I would prefer to keep the `Option` to signal the error condition rather than silently hide it, what do you think @Kha?

Reply:
I can make it return Option string then, not a problem.  I think the future full Uri parser will do things differently, and we can change it then, for example "/tmp/test.lean" is a valid Uri, it just has the "scheme" of "".  Then your code would check the scheme is "file" and proceed from there with the Uri.path component, etc...

Reply:
After all that I found this code in the LSP to be pretty funny `doc.uri.toPath?.getD doc.uri`

Reply:
and I did the "rm" operation to remove the redundant methods.

Reply:
Hah even in the other places if it's `none` we just ignore it rather than flag an error 😅

Reply:
indeed, but at least there we have a different control flow happening in the none case.